saas/api/metrics.py

```diff
@@ -22,13 +22,17 @@
 # OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
 # ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from django.core.serializers.json import DjangoJSONEncoder
 from django.db.models import Q
+from django.utils.datastructures import SortedDict
 from django.utils.dateparse import parse_datetime
+import json
```

Comment:
This import turns out to be vestigal, anyway. I'm removing it. But, thanks for the clarification on the import order convention!